source/compiler/sc.h

```diff
@@ -640,7 +640,7 @@ SC_FUNC void setlinedirect(int line);
 SC_FUNC void setlineconst(int line);
 SC_FUNC void setlabel(int index);
 SC_FUNC void markexpr(optmark type,const char *name,cell offset);
-SC_FUNC void startfunc(char *fname);
+SC_FUNC void startfunc(char *fname, char isnaked);
```

Comment:
The compiler codebase usually utilizes `int` for boolean variables, not `char`.

Reply:
The styling used in the project requires that there isn't a space between arguments.

Reply:
I am being evil but I think the parameter makes more sense if it is related to the prologue than nakedness of a function?